src/aggregate/aggregate.h

```diff
@@ -220,7 +234,8 @@ typedef struct AREQ {
   /** Flags indicating current execution state */
   uint32_t stateflags;
 
-  int protocol; // RESP2/3
+  /** Protocol version RESP2/3 */
+  unsigned char protocol;
```

Comment:
Nitpick: might affect memory alignment? (depending on the compiler)
Not sure it has significant influence.
If needed for correctness, we can keep it.

Reply:
Reverted.